src/layout/linear/mod.rs

```diff
@@ -222,6 +222,20 @@ where
     }
 }
 
+impl<LD, VG> Clone for LinearLayout<LD, VG>
+where
+    LD: Orientation,
+    VG: ViewGroup + Clone,
+{
+    fn clone(&self) -> Self {
+        Self {
+            position: self.position.clone(),
+            direction: self.direction.clone(),
```

Comment:
```suggestion
            position: self.position,
            direction: self.direction,
```

I think both `Point` and `Orientation` are `Copy` so there is no need to call `clone()` on them.

Reply:
Moved into another PR